.dependabot/config.yml

```diff
@@ -0,0 +1,5 @@
+version: 1
+update_configs:
+  - package_manager: "java:gradle"
```

Comment:
probably nice to get this in separate PR :)

Reply:
meant to put it in my update dependencies PR to fix the owasp errors oops

Reply:
will get the other one in first